runelite-client/src/main/java/net/runelite/client/RuneLite.java

```diff
@@ -77,6 +77,7 @@
 	public static final File RUNELITE_DIR = new File(System.getProperty("user.home"), ".runelite");
 	public static final File PROFILES_DIR = new File(RUNELITE_DIR, "profiles");
 	public static final File SCREENSHOT_DIR = new File(RUNELITE_DIR, "screenshots");
+	public static final Locale SYSTEM_LOCALE = Locale.getDefault();
```

Comment:
This is unsused

Reply:
Its still used for non-windows formatter

Reply:
oh.
This shouldn't exist at all; any users of it should just call Locale.getDefault().

Reply:
Well, then it would be always US because of our Main method "hack" where we force set default locale

Reply:
then the place where we set the locale to us should set it to english, not us

Reply:
Oops, its actually set to ENGLISH not us
```
		Locale.setDefault(Locale.ENGLISH);
```

but point of this change is so the formatter respects system locale, and on linux switching between locales is trivial (e.g I want AM/PM, I just change locale to US, I want 24hr, I change to UK).

Reply:
Most OSes have proper locale support, so that isn't reasonable for most users because that changes many other things, such as decimal formatting, spell checking/autocorrect, etc.

Reply:
Well, that is my use-case. But still, it should respect system locale, as there is nothing like AM/PM switch on Linux (idk on Mac). And most people just prefer their system locale date format, so I dont think this is an issue. You have your windows switch, and I want my Linux switch.

Reply:
the default locale should be affected by the switch. We shouldn't have to maintain 2 locales for different things

Reply:
It is the only way to get AM/PM setting on Linux (and as I already said, that is the point of this PR), so yes we do. and using static variable is less hacky than using WinAPI on Windows, so I think it is fine?

Reply:
having 2 static variables is worse than 1. we can just have 1 and be done with it. you said that it should already work without this

Reply:
Without this it will always use Locale.ENGLISH instead of user-set locale that will determine AM/PM. ENGLISH is UK afaik, so it would be always 24hr if we would use Locale.getDefault() instead of this in StackFormatter

Reply:
```
Locale.setDefault(new Locale.Builder()
	.setLocale(Locale.getDefault())
	.setLanguageTag("en")
	.build());
```

Reply:
Hmmm, will that actually work? I guess tag is first part of this

```
export LANG="en_US.UTF-8"
export LC_ALL="en_US.UTF-8"
```

so it might.